### ISSUE TITLE
Is Arduino Mega 2560 controller supported?

### ISSUE THREAD

**Issue by apodavalov** [MegaPirateNG/ardupilot-mpng#123]
Is Arduino Mega 2560 controller supported? Or which version of MegaPirateNG I have to use? 2.x from the old site?

**Comment by smurfy** [comment]
What flight controller are you using?
All Atmel based flight controllers supported by MPNG using the 2560 Atmel CPU.

If you mean the Arduino Mega 2560 development board, then yes you can flash MPNG on it, but you will get problems, because you also need the different sensors.

**Comment by apodavalov** [comment]
Yes, I'm using Arduino Mega 2560 development board with ALLINONE sensors.

**Comment by smurfy** [comment]
Try it, i successfully flashed R4 to my Arduino Mega, but it hang while booting up the software while initialisation. Not sure if it's due to the missing sensors or another problem.

I would suggest using a standalone flight controller anyway, because of smaller size and weight.
